doc/modules/changes/20191013_naliboff

```diff
@@ -0,0 +1,6 @@
+Changed: The viscoelastic plastic material has been reordered so
+that the viscoelastic viscosity is calculated prior to yielding
+and viscoelastic stresses are taken into account when comparing
+the current stress against the plastic yield stress. 
```

Comment:
Could you add the comment that this is now consistent with Louis's paper that you cite in the code?